distribution/docs/src/main/resources/content/_configuring/http-port.adoc

```diff
@@ -1,17 +1,23 @@
-:title: Configuring HTTP Port
+:title: Configuring HTTP(s) Ports
 :type: configuration
 :status: published
 :parent: Configuring Federation
-:summary: Configure HTTP Port
+:summary: Configure HTTP(s) Port.
 :order: 01
 
 == {title}
 
+To change HTTP or HTTPS ports from the default values, edit the `system.properties` file.
+
+. Open the file at ${home_directory}/etc/system.properties
+. Change the value after the `=` to the desired port number(s):
+.. `org.codice.ddf.system.httpsPort=8993` to `org.codice.ddf.system.httpsPort=<PORT>`
```

Comment:
❓ Is there a alias for `org.codice.ddf` like there is for branding?

Reply:
currently, no